### ISSUE TITLE
Build page template for front page

### ISSUE THREAD

**Issue by SDavisMedia** [easydigitaldownloads/vendd#216]
Nope, it _will not_ be a front-page.php template. But it'll be a Page Template intended to be used on front pages, though it can be used anywhere.

It'll most likely depend on some 3rd party plugins to output things like grid display of features with icons, testimonials, etc. And of course, a way to display some EDD downloads. It'll just be a nice tool for users to present an overview of the site.

It's _not_ going to be in Vendd 1.2. 🙃

**Comment by davidsherlock** [comment]
@SDavisMedia, I am actually doing this with the theme I am building and trying to utilize Jetpack  for all the testimonial handling, portfolios, sharing, etc. I want to be able to build any type of home page, here are some screenshots of my progress so far (all done via shortcodes and/or custom widgets i have incorporated via theme locations).

Jetpack testimonials shortcode styling

![screenshot 2017-05-25 22 06 42](https://cloud.githubusercontent.com/assets/22668877/26470666/74ca0596-4196-11e7-8689-f590a02ce812.png)

Featured widget (added various widget locations to my home page template:

![screenshot 2017-05-25 22 06 30](https://cloud.githubusercontent.com/assets/22668877/26470695/85c0d5b4-4196-11e7-9758-a911d30e958b.png)

Pricing tables powered by EDD variable priced options.

![screenshot 2017-05-25 22 06 18](https://cloud.githubusercontent.com/assets/22668877/26470708/95104144-4196-11e7-9664-09e182c18e2d.png)

Sharing buttons:

![screenshot 2017-05-25 22 08 26](https://cloud.githubusercontent.com/assets/22668877/26470752/b587e396-4196-11e7-8c89-48e26934dc1b.png)

I still have to add latest articles, CTA section and team members. All of which can be added to the home page, along with any type of post content. I hate page builders, so attempting all of this in as clean a fashion as possible.

**Comment by SDavisMedia** [comment]
@davidsherlock yes that's _exactly_ what I'll be going for. Basically a way for users to fill in a data structure that already exists... as in, they either like it (and use it) or they don't. Nothing too customizable in regard to design... at least not from the customizer. They can always override template files, of course. What you're doing is right on the money.

**Comment by davidsherlock** [comment]
Slightly off-topic @SDavisMedia, but it would be cool to work together on a commercial EDD theme at some point! Feel we are on the same page with design decisions, code, etc.

**Comment by SDavisMedia** [comment]
@davidsherlock I'm totally for that! I hope to start building another later this year. Perhaps we can link up then.

**Comment by sumobi** [comment]
Great timing, I started building "EDD Pricing Tables" earlier this week that we can all use in our themes. @davidsherlock Like you, I went the route of just using the download's existing variable pricing, but you can also pick separate downloads to list in the table if need be. Super simple, since all the existing information is based on the download. No new/ugly UI to manage pricing tables (I mean seriously, how often do you need to build them?). Can't wait to replace the custom pricing tables on AffiliateWP and Restrict Content Pro

I spent hours last night pondering over building the "ideal" homepage layout, glad to see this issue @SDavisMedia. For Themedd I'm currently leaning towards defining custom homepage sections like the Twenty Seventeen theme. This would allow the user to create a page, chuck the [downloads] shortcode on it and output via one of the custom sections, as well as adding any other pages they need. Still thinking it through...

**Comment by SDavisMedia** [comment]
@sumobi yea I definitely want something that can function outside of just variable pricing. In EDD support, I've seen many people want to use EDD custom add to cart links as the purchase buttons on pricing tables. So I imagine something flexible enough to go from variable pricing to custom bullet points/call-to-action without much hassle.

**Comment by sumobi** [comment]
Yep, uses custom add to cart links by default and a textarea field to enter features per pricing option (which are then outputted on different lines). Will support stripe checkout and all that jazz.

**Comment by davidsherlock** [comment]
@SDavisMedia, do you plan to make a sectioned homepage (or page template)?

**Comment by SDavisMedia** [comment]
@davidsherlock yes but don't wait on it. If you need it, go ahead and build it.
